doc/system_specific_setup.rst

```diff
@@ -103,6 +103,19 @@ may be downloaded from https://github.com/stfc/fparser/releases. Simply
 unpack the tarball and ensure that the resulting
 ``fparser-x.y.z/src/fparser`` directory is in your PYTHONPATH.
 
+PSyclone supports the ability to output a schedule dependency graph
+using the graphviz package. This is optional and the associated
+routine will silently return if the graphviz bindings are not
+installed. To output a graph you need to install the graphviz package
+::
+
+   > sudo apt-get install graphviz
```

Comment:
Yes it is but that is on purpose as we are in the Ubuntu-specific set up section which supports apt